cmis_web_proxy/README.rst

```diff
@@ -0,0 +1,78 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Web Cmis Viewer Proxy
+=====================
+
+Odoo as proxy server for your cmis requests from the web to the cmis container.
+
+Installation
```

Comment:
For sure this readme should be improved...

Reply:
Can you adapt the readme to look like https://raw.githubusercontent.com/acsone/alfodoo/10.0/cmis_web/README.rst